### PR TITLE
AFK players now show up as SSD on examine (+ soft ping port)

### PR DESCRIPTION
**Pull request by Absolucy**
## About The Pull Request

This makes it so that AFK players will appear as SSD. This uses the inactivity period configuration, which is 5 minutes by default.

Although perhaps a different message could be shown for AFK players as opposed to SSD players? I'm open to suggestions, I'm not that good at coming up with flavor text.

Also ports https://github.com/tgstation/tgstation/pull/66299, which fixes inactivity/`is_afk` not working, as uh, that kinda needs to work for this.

## Why It's Good For The Game

Makes it less likely for me to be killed when I'm AFK (but not disconnected) because I had to go use the bathroom or do some chores, due to someone not being able to tell I'm AFK when examining me.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

## Soft-ping
image

## AFK examine (+ working `is_afk`)
image

</details>

## Changelog
:cl: Absolucy, stylemistake
add: AFK players will now have flavor text indicating that they're AFK when examined.
fix: AFK detection now works properly again.
/:cl:

### CODE REVIEW

**Comment by PowerfulBacon** [comment]
Why would we want to let players adjust how long until they are registered as being AFK? This should be a config setting and set entirely by the server.

**Comment by Absolucy** [comment]
Ah, fair. I'm not sure why I thought of that.

Should it be a separate config option or just the `INACTIVITY_PERIOD` config?

**Comment by PowerfulBacon** [comment]
oh yea, inactivity period is probably fine

**Comment by Absolucy** [comment]
Aight, I changed it to just use the default `is_afk` arguments, which is the inactivity period configured by the server.

**Comment by itsmeow** [comment]
This should also show on MedHUDs

**Comment by Absolucy** [comment]
@itsmeow like as in the "dead, distant soul" medhud?

**Comment by itsmeow** [comment]
the yellow question mark

**Comment by Absolucy** [comment]
@itsmeow done

File: code/controllers/subsystem/ping.dm [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+SUBSYSTEM_DEF(ping)
+	name = "Ping"
+	priority = FIRE_PRIORITY_PING
+	wait = 4 SECONDS
+	flags = SS_NO_INIT
+	runlevels = RUNLEVEL_LOBBY | RUNLEVELS_DEFAULT
+	var/list/currentrun = list()
+
+/datum/controller/subsystem/ping/stat_entry()
+	..("P:[length(GLOB.clients)]")
```
Comment (PowerfulBacon) [inline code/controllers/subsystem/ping.dm]: This function needs to return a value

**Review by PowerfulBacon — DISMISSED** [review]

**Comment by itsmeow** [comment]
You have to actually call medhud update status when the afk value changes or it won't actually show.

**Comment by Absolucy** [comment]
@itsmeow What'd be the best way to handle that? Since `is_afk` is polled, rather than being constantly checked...

**Review by itsmeow — APPROVED** [review]
Looks good.

**Pull request merged**